game_patch/multi/server.cpp

```diff
@@ -554,6 +554,20 @@ static bool check_player_ac_status([[maybe_unused]] rf::Player* player)
     return true;
 }
 
+std::set<rf::Player*> get_current_player_list(bool include_browsers)
```

Comment:
I designed the function to be flexible for use in the future in cases where browsers perhaps should be included. I don't have a use case for that currently, but the added overhead is worth the flexibility in my mind. Happy to adjust the PR if you disagree with that, though.

"current" doesn't serve a purpose, good point. I'll adjust that.

I use this function in other places in my code - the match mode ready system for example - and perhaps other places in the future, because of that I put it in server.cpp because that seemed to be where it would best fit.

Reply:
Why is it a set? Why should browsers ever be allowed to vote? I do not like it.

Reply:
Browsers should never be allowed to vote - this function isn't intended to be used only as a component of voting.

Reply:
As I said I don't see why would anybody call `get_player_list(true)` if they can just use `rf::player_list` directly. The only thing this function adds to the table is filtering out browsers. So it should be named accordingly and probably placed in a different place. For now the simplest way forward is to put it into `votes.cpp` module as a local helper function, because we don't know where it will be useful in the future (and if it will).
I wonder if it works on the client-side. It could skip browsers in spectate mode